src/lib/rest/rest.cpp

```diff
@@ -82,6 +83,20 @@ static unsigned int              threadPoolSize;
 
 /* ****************************************************************************
 *
+* correlatorGenerate - 
+*/
+static void correlatorGenerate(char* buffer)
```

Comment:
It is a thread variable, so that should not be a problem.
But you are probably thinking of the functions from libuuid.
Good question, needs some research.